apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/HiPerfSubscriber.cs

```diff
@@ -462,55 +485,70 @@ public Flow(long maxByteCount, long maxElementCount, Action<Task> registerTaskFn
                     {
                         if (IsFlowOk())
                         {
+                            // Flow is OK, so add for this element, and execute.
                             _byteCount += byteCount;
                             _elementCount += 1;
                             break;
                         }
+                        // FLow not OK, prepare to wait until a previous fn completes.
```

Comment:
Nit: FLow => Flow

(But these comments are making it *so* much easier to understand...)